lib/ddtrace/contrib/sidekiq/configuration/settings.rb

```diff
@@ -17,6 +17,11 @@ class Settings < Contrib::Configuration::Settings
             o.lazy
           end
 
+          option :tag_args do |o|
+            o.default { env_to_bool(Ext::ENV_TAG_JOB_ARGUMENTS, false) }
```

Comment:
Can we also abbreviate everywhere `arguments` appears as `args` instead? For consistency. Otherwise this looks good to merge to me.

Reply:
@delner done!